deploy_django.yml

```diff
@@ -1,26 +1,25 @@
 ---
 
 - name: Deploy Django Application to Web Servers
-  hosts: tag_Name_ansible-built
+  hosts: tag_Name_django-imager
   remote_user: ubuntu
   gather_facts: yes
-  
+
   roles:
     - role: django
     - role: gunicorn
     - role: jdauphant.nginx
       sudo: yes
-      nginx_sites: 
+      nginx_sites:
         default:
           - listen 80
           - server_name "{{ ec2_public_dns_name }}"
           - access_log "/var/log/nginx/django.log"
-          - location / { 
+          - location / {
               proxy_pass http://127.0.0.1:8000;
               proxy_set_header Host $host;
               proxy_set_header X-Real-IP $remote_addr;
               proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
             }
           - location /static/ { root /var/www/django-imager; }
-          - location /media/ { root /var/www/django-imager; }
-
+          - location /media/ { root {{ django_home }}/imagersite; }
```

Comment:
Similarly, django_home/imagersite should itself also be a variable.

The idea is to make your playbooks as abstract and re-usable as possible, without any project-specific names, folders, etc.